ssh/tailssh/tailssh.go

```diff
@@ -514,34 +533,6 @@ func (c *conn) mayForwardLocalPortTo(ctx ssh.Context, destinationHost string, de
 	return false
 }
 
-// havePubKeyPolicy reports whether any policy rule may provide access by means
```

Comment:
Because we now need to check for a welcome banner before we've even started authentication, we can't base anything on the client-supplied public key. Since public key policies aren't yet implemented in control and not yet used, I removed this. If and when we want to do this, we can look at adding something back in, potentially needing new upstream changes to golang.org/x/crypto.